bindgen/ir/comp.rs

```diff
@@ -595,42 +606,41 @@ where
         let bitfield_width = bitfield.bitfield_width().unwrap() as usize;
         let bitfield_layout =
             ctx.resolve_type(bitfield.ty()).layout(ctx).ok_or(())?;
-        let bitfield_size = bitfield_layout.size;
         let bitfield_align = bitfield_layout.align;
+        // bitfield offset of struct/union
+        let bitfield_offset = bitfield.offset().unwrap();
```

Comment:
Actually I'm not sure we can guarantee we have this, right? This panics with this patch:

```
template <typename T> struct foo {
  T member;
  bool b : 8;
};
```